lib/puppet/util/metric.rb

```diff
@@ -26,7 +26,7 @@ def to_data_hash
     {
       'name' => @name,
       'label' => @label,
-      'values' => @values
+      'values' => @values.map { |v| [v[0], v[1], v[2].round(3)] if v[2].is_a?(Float) }
```

Comment:
@joshcooper updated

Reply:
hmm according to https://tools.ietf.org/html/rfc7159.html#section-6, exponential values are allowed in the JSON spec...

Reply:
Also according to https://spacetelescope.github.io/understanding-json-schema/reference/numeric.html#number, "In Ruby, “number” is analogous to the Float type." So I think we can assume `v[2]` is a Float (or Integer or Fixnum in older rubies) and we should be able to call `round` always:

In 1.9.3:

```
irb(main):001:0> 0.class
=> Fixnum
irb(main):002:0> 0.round(6)
=> 0.0
```

or in 2.4.1:

```
(byebug) 0.class
Integer
(byebug) 0.round(6)
0.0
```

Reply:
So it will never be a string?
```
irb(main):003:0> 'string'.class
=> String
irb(main):004:0> 'string'.round(3)
NoMethodError: undefined method `round' for "string":String
        from (irb):4
        from /Users/melissa/.rbenv/versions/2.4.0/bin/irb:11:in `<main>'
```